main.py

```diff
@@ -195,21 +186,21 @@ def delete_images(ecr_client, deletesha, deletetag, id, name):
 
 # Below is the test harness
 if __name__ == '__main__':
-    request = {"None": "None"}
-    parser = argparse.ArgumentParser(description='Deletes stale ECR images')
-    parser.add_argument('-dryrun', help='Prints the repository to be deleted without deleting them', default='true',
+    REQUEST = {"None": "None"}
```

Comment:
Can you explain why these variables are uppercase now?

Reply:
It was a Pylint suggestion, since they're outside a function they're global constants.

Reply:
I don't have a preference either way 🤷‍♂ I'm not familiar with python and was mostly just curious

Reply:
In Python, files act like modules/packages/namespaces so Pylint suggests that although it sometimes leads to weird syntax

For example take `example.py` like
```python
SOME_VAR = "something"
```
and `some_script.py`
```python
import example
print(example.SOME_VAR)
```

if you left it lowercase, it would look like a method if you imported it somewhere else hence the suggestion to put it in uppercase. In this case, those variables only exist if the file is being ran directly, not imported so it probably doesn't really matter. You could always just dump it in a method then just call the method